src/extension/background-script/actions/nostr/signEventOrPrompt.ts

```diff
@@ -17,15 +19,44 @@ const signEventOrPrompt = async (message: MessageSignEvent) => {
     };
   }
 
+  const allowance = await db.allowances.get({
+    host: message.origin.host,
+  });
+
+  if (!allowance?.id) {
+    return { error: "Could not find an allowance for this host" };
+  }
+
+  const findPermission = await db.permissions.get({
+    host: message.origin.host,
+    method: PermissionMethod["NOSTR_SIGNMESSAGE"],
+  });
+
+  const hasPermission = !!findPermission?.enabled;
```

Comment:
https://stackoverflow.com/questions/53060324/using-a-field-with-a-boolean-type-in-dexie-js
Can't use boolean in `get/where`?